src/layouts/header/index.tsx

```diff
@@ -20,10 +21,19 @@ type Props = {
 };
 
 const Header = ({ className }: Props) => {
+  const [isSignatured, setIsSignatured] = useState(false);
   const sticky = useSticky();
   const { offcanvas, offcanvasHandler } = useOffcanvas();
   const { search, searchHandler } = useFlyoutSearch();
-  const { connect, connected } = useMetamask();
+  const { connect, connected, sign } = useMetamask();
+
+  useEffect(() => {
+    if (typeof window !== undefined) {
```

Comment:
You don't need this. There is a auth slice where you can get the values you need

Reply:
I need to get the authToken from the localStorage instead the redux store

Reply:
Hmm, according to your code, all you need is to know if there is a token in local storage. If the auth slice value 'isAuthenticated' is true, it means that it has valid token which is better yet